range.c

```diff
@@ -2651,6 +2652,16 @@ range_overlap(VALUE range, VALUE other)
  * - #to_a (aliased as #entries): Returns elements of +self+ in an array.
  * - #to_s: Returns a string representation of +self+ (uses #to_s).
  *
+ * === Methods for Working with JSON
+ *
+ * To make these methods available:
+ *
+ *   require 'json/add/range'
```

Comment:
I think this is an internal implementation detail, we should just say that they need to load all of JSON.

```suggestion
 *   require 'json'
```